tesla_ble_mqtt/config.yaml

```diff
@@ -17,33 +17,30 @@ map:
   - share:rw
 startup: services
 options:
-  vin1: ""
-  vin2: ""
-  vin3: ""
-  presence_detection: false
-  ble_mac1: ""
-  ble_mac2: ""
-  ble_mac3: ""
+  vin: ""
+  vin_list: ""
+  ble_mac: ""
+  ble_mac_list: ""
+  presence_detection_ttl: 180
   debug: false
   mqtt_ip: ""
   mqtt_port: "1883"
   mqtt_user: ""
   mqtt_pwd: ""
   send_cmd_retry_delay: "5"
+
 schema:
-  vin1: str?
-  vin2: str?
-  vin3: str?
-  presence_detection: bool
-  ble_mac1: str?
-  ble_mac2: str?
-  ble_mac3: str?
+  vin: str?
+  vin_list: "match(^([A-HJ-NPR-Z0-9]{17})(\|[A-HJ-NPR-Z0-9]{17})*$)"
+  ble_mac:str?
+  ble_mac_list: "match(^([0-9A-Fa-f]{2}(:[0-9A-Fa-f]{2}){5})(\|[0-9A-Fa-f]{2}(:[0-9A-Fa-f]{2}){5})*$)?"
+  presence_detection_ttl: "int(0,)"
   debug: bool
-  mqtt_ip: str?
+  mqtt_ip: str
   mqtt_port: str?
   mqtt_user: str?
```

Comment:
Teslamate uses USERNAME ; I am a user of this computer but a username is a username.

Reply:
Renamed to _USERNAME